CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.0.11

```

Comment:
this should not be uplifted, we haven't released 2.0.10 yet. 

Add this to 2.0.10